changelog/fragments/predicateChanges.yaml

```diff
@@ -0,0 +1,12 @@
+entries:
+  - description: >
+      Added predicate filtering function for labels based on selectors specified in watches.yaml. 
+      Added temporary structs to watches.go for yaml parsing in UnmarshalYAML
```

Comment:
line 4 is kinda implementation detail not meant to surface as user facing change logs?

Reply:
Yeah, what you should put here is a brief description of the feature from the user's perspective, no need to talk about the code base directly